tests/default_vars.sh

```diff
@@ -284,94 +331,121 @@ export_cpl ()
 {
 export S2S=true
 
+export DAYS="1"
+export FHMAX="24"
+export FDIAG="6"
+export WLCLK=30
+
+# default atm/ocn/ice resolution
+export ATMRES='C96'
+export OCNRES='100'
+export ICERES='1.00'
+export NX_GLB=360
+export NY_GLB=320
+
+# default resources
 export TASKS=$TASKS_cpl_dflt
 export TPN=$TPN_cpl_dflt
 export INPES=$INPES_cpl_dflt
 export JNPES=$JNPES_cpl_dflt
 export THRD=$THRD_cpl_dflt
-
 export WRTTASK_PER_GROUP=$WPG_cpl_dflt
+
 export med_petlist_bounds=$MPB_cpl_dflt
 export atm_petlist_bounds=$APB_cpl_dflt
 export ocn_petlist_bounds=$OPB_cpl_dflt
 export ice_petlist_bounds=$IPB_cpl_dflt
 
-export ATMRES='C96'
-# default ice and ocean resolution
-export OCNRES='025'
-export ICERES='0.25'
-export NX_GLB=1440
-export NY_GLB=1080
-export SUITE_NAME=''
-export MED_restart_data=''
-export INPUT_NML="input.mom6.nml.IN"
+# component and coupling timesteps
+export DT_ATMOS='900'
+export DT_CICE=${DT_ATMOS}
+export DT_DYNAM_MOM6='1800'
+export DT_THERM_MOM6='3600'
+export CPL_SLOW=${DT_THERM_MOM6}
+export CPL_FAST=${DT_ATMOS}
+
+# nems.configure defaults
+export NEMS_CONFIGURE="nems.configure.medcmeps_atm_ocn_ice.IN"
```

Comment:
Wondering if this would be the same as `nems.configure.medcmeps_cpld.IN` or just `nems.configure.cpld.IN`?

Reply:
I agree. I would suggest we rename the nems.configure().IN to nems.configure.cpld.IN and nems.configure.cpld.wave.IN